src/pg/connection.cr

```diff
@@ -37,6 +37,20 @@ module PG
       end
     end
 
+    # `#initialize` Connect to the server with values of Hash.
+    #
+    # ```
+    # info = { host: "localhost", user: "postgres", password: "password", db_name: "test_db", port: "5432"}
+    # PG::Connection.new(info)
+    # ```
+    def initialize(conninfo : Hash)
```

Comment:
I think 
`parameters : YAML::Type => YAML::Type`
is better, because sometime connect with using yaml file.